aliasmgr/aliasmgr.go

```diff
@@ -10,6 +10,11 @@ import (
 	"github.com/lightningnetwork/lnd/lnwire"
 )
 
+// UpdateLinkAliases locates the active link that matches the given
+// shortID and triggers an update based on the latest values of the
+// alias manager.
+type UpdateLinkAliases func(shortID lnwire.ShortChannelID) error
```

Comment:
This commit needs to come second to fix the CI (the RPC commit won't compile without this).